local/artena/externallib.php

```diff
@@ -447,11 +472,11 @@ public static function remove_course($courses) {
                 $existing_course = $DB->get_record('course', array('fullname' => $course['fullname'], 'shortname' => $course['shortname']));
             }
             if (false === $existing_course) {   // unknown course
-                throw new invalid_parameter_exception('Unknown course: '.$assignment['idnumber']);
+                throw new invalid_parameter_exception('Unknown course: '.$course['idnumber']);
```

Comment:
Unsure if this is correct $assignment could be $course? Haven't had any trouble.